tutorial/stdlib.po

```diff
@@ -35,25 +35,24 @@ msgid ""
 "This will keep :func:`os.open` from shadowing the built-in :func:`open` "
 "function which operates much differently."
 msgstr ""
-"Mais, encore une fois, préférez ``import os``, à ``from os import *``, sans "
-"quoi :func:`os.open` cacherait la primitive :func:`open`, qui fonctionne "
-"différemment."
+"Veillez bien à utiliser ``import os`` plutôt que ``from os import *``, sinon :"
+"func:`os.open` cache la primitive :func:`open` qui fonctionne différemment."
 
 #: ../Doc/tutorial/stdlib.rst:29
 msgid ""
 "The built-in :func:`dir` and :func:`help` functions are useful as "
 "interactive aids for working with large modules like :mod:`os`::"
 msgstr ""
-"Les primitives :func:`dir` et :func:`help` sont des outils utiles lorsque "
-"vous travaillez en mode interactif avez des gros modules comme :mod:`os` ::"
+"Les primitives :func:`dir` et :func:`help` sont des aides utiles lorsque vous "
+"travaillez en mode interactif avez des gros modules comme :mod:`os` : ::"
 
 #: ../Doc/tutorial/stdlib.rst:38
 msgid ""
 "For daily file and directory management tasks, the :mod:`shutil` module "
 "provides a higher level interface that is easier to use::"
 msgstr ""
 "Pour la gestion des fichiers et dossiers, le module :mod:`shutil` expose une "
-"interface plus abstraite et plus facile à utiliser : ::"
+"interface plus abstraite et plus faciles à utiliser : ::"
```

Comment:
As the subject is *Une interface* why put a plural sign on *facile* ?

Reply:
Oui, mauvaise manipulation en corrigeant le 2e "facile" plus loin dans le texte.
Merci